app/src/main/java/org/p2p/wallet/striga/wallet/models/StrigaBlockchainNetworkInfo.kt

```diff
@@ -0,0 +1,11 @@
+package org.p2p.wallet.striga.wallet.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class StrigaBlockchainNetworkInfo(
+    val name: String,
```

Comment:
It's not really a network name, because it might contain weird stuff like: `USD Coin Test (Goerli)`.
Fancy that it doesn't represent the network name, real token name from the contract, or even token type.